app/src/main/java/com/netmera/netmerafintech/utils/AnalyticUtils.kt

```diff
@@ -0,0 +1,37 @@
+package com.netmera.netmerafintech.utils
+
+import android.app.Activity
+import android.os.Build
+import com.netmera.Netmera
+import com.netmera.netmerafintech.Netmera.Events.SignInEvent
+import com.netmera.netmerafintech.Netmera.Events.GetStartedEvent
+import com.netmera.netmerafintech.data.model.NMImpactFintechUser
+
+object AnalyticsUtil {
+
+    fun getNotificationPermission(activity: Activity) {
+        if (Build.VERSION.SDK_INT >= 33) {
+            Netmera.requestNotificationPermissions(activity)
+        } // already granted for lower level
+    }
+
+    fun getStartedEvent() {
+        val event = GetStartedEvent()
+        event.setGetStarted(true)
+        Netmera.sendEvent(event)
+    }
+
+    fun updateNetmeraUser() {
+        val netmerauser = NMImpactFintechUser()
+        netmerauser.setUserId("burakaymaz@hotmail.com")
+        netmerauser.setName("Burak")
+        netmerauser.setType("black")
```

Comment:
This is offensive! Whose idea was this :D

Reply:
Burak Kaymaz instructed it.

Reply:
I was joking. No problem :)